sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/SearchDocumentsResult.java

```diff
@@ -54,6 +55,16 @@ public final class SearchDocumentsResult implements JsonSerializable<SearchDocum
      */
     private SearchRequest nextPageParameters;
 
+    /*
+     * Reason that a partial response was returned for a semantic search request.
+     */
+    private SemanticPartialResponseReason semanticPartialResponseReason;
```

Comment:
These values will need to be added into `SearchPagedFlux` and `SearchPagedIterable` in the same way `coverage`, `totalCount`, and `answers` are handled.